common/views/components/SearchTabs/SearchTabs.tsx

```diff
@@ -2,27 +2,20 @@ import BaseTabs, { TabType } from '../BaseTabs/BaseTabs';
 import { classNames, font } from '@weco/common/utils/classnames';
 import styled from 'styled-components';
 import Space from '../styled/Space';
-import { useContext, FunctionComponent, ReactElement } from 'react';
+import React, { useContext, FunctionComponent, ReactElement } from 'react';
```

Comment:
Couldn't see where/how you were using `React` in this file

Reply:
removed